### DISCUSSION TITLE
Greeting, As I'm a fresh web developer, I'd like to make Dynamic portfolio website. Help me!!!

### DISCUSSION THREAD

**Discussion by MYong0109** [community/community#29808] [Other Feature Feedback, Questions, & Ideas] [answered]
Before 5 days ago, I'm trying to make portfolio with Babylon.js.

But, I have faced some issue for integrating React framework.

Please help me!!

**Comment by phantom0109** [comment, marked as answer]
In my opinion, you can use three.js fiber/drei.

I have studied three.js from a year ago.

The three.js have some module for integrating front-end(exactly react, next.js), So I have used three.js-react module.

You have to know about 3D coordinate and 3D graphic environment.

If you have any questions, please don't hesitate to call me.

Always ready.
